lessons/07-html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<html>
+    <title>Create User</title>
+    <header>
+        <h1>Create User</h1>
+    </header>
+    <hr />
+    <form action="create-user.html" method="get">
+            <label for="email">E-mail:</label>
+            <input type="email" name="email" id="email" placeholder="email" /><br /><br />
+            <label for="username">Username:</label>
+            <input type="text" name="username" id="username" size="30" maxlength="12" placeholder="username" /><br /><br />
+            <label for="password">Password:</label>
+            <input type="password" name="password" id="password" size="30" maxlength="8" placeholder="password" /><br /><br />
+            <label for="bio">Bio:</label>
+            <textarea name="bio" id="bio" cols="30" rows="2" placeholder="give a brief description of the user"></textarea><br /><br />
```

Comment:
We need to reserve using `<br />` only for when we want a new line inside of a paragraph.

What we can do here instead is group the inputs + labels together with `<div>` tags

```html
<div>
  <label for="username">User name : </label>
  <input type="text" name="username" placeholder="username" size="30" maxlength="12">
</div>

<div>
  <label>Password:</label>
  <input type="password" name="password" placeholder="password" size="30" maxlength="8" required/>
</div>
```